CommandLine/emake/Makefile

```diff
@@ -16,18 +15,23 @@ else
 	OS_LIBS=-lboost_system-mt -Wl,--no-as-needed -Wl,-rpath,./ -lboost_program_options-mt -lboost_iostreams-mt -lboost_filesystem-mt -lboost_system-mt -lpthread
 endif
 
-PROTO_DIR := $(SHARED_SRC_DIR)/protos
-CXXFLAGS  := -I$(SRC_DIR) -I../../CompilerSource -I$(SHARED_SRC_DIR) -I$(PROTO_DIR)/.eobjs -std=c++11 -Wall -Wextra -Wpedantic -g
-LDFLAGS   := $(OS_LIBS) -L../../ -lProtocols -lprotobuf
+CXXFLAGS  += -I../../CompilerSource -I$(PROTO_DIR)
+LDFLAGS   += $(OS_LIBS) -L../../ -lProtocols -lprotobuf
 
-rwildcard=$(wildcard $1/$2) $(foreach d,$(wildcard $1/*),$(call rwildcard,$d,$2))
-SOURCES := $(call rwildcard,$(SRC_DIR),*.cpp)
+ifeq ($(TESTS), TRUE)
```

Comment:
Losing TARGET would also let us just build these two as targets instead of using `TEST=true $(MAKE) ...`

Reply:
Well we still build them as targets from the main makefile and thats the only one anyone should be calling really